src/WinXP/apps/InfoDialog/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+
+import Button from 'components/Button';
+import info_bubble from 'assets/info-bubble.png';
+
+function InfoDialog({ info, onClose }) {
+  const windowRef = useRef(null);
+  useEffect(() => {
+    /// Forcibly focus on the window
+    windowRef.current.focus();
+  }, []);
+
+  return (
+    <InnerWindow
+      tabIndex="-1"
+      ref={windowRef}
+      onKeyDown={e => (e.key === 'Enter' || e.key === 'Escape') && onClose()}
```

Comment:
@RonnyO I checked a few days ago and this behavior did exist in the find, replace, info dialog windows. But I checked again just now and it seems not to be working- I believe it is caused by the virtual box issues, because typing inside the notepad, keyboard navigation etc. also does not work currently